### ISSUE TITLE
When it finds an insertion with several new lines it does not assign the corresponding attributes to the <p> tag

### ISSUE THREAD

**Issue by CatHood0** [VisualSystemsCorp/vsc_quill_delta_to_html#17]
#### Input 

`{insert: Should justify the text}, {insert: \n\n, attributes: {align: justify}}`

(Should be different, but, now Flutter Quill use this format by some reason)

#### output:
```html
<p>Should justify the text</p>
<p><br><p>
<p style="text-align: justify"><br></p>
```
#### expected
```html
<p style="text-align: justify">Should justify the text</p>
<p style="text-align: justify"><br><p> or <p><br><p>
```

#### Converter options
```dart
 ConverterOptions(
          orderedListTag: 'ol',
          bulletListTag: 'ul',
          multiLineBlockquote: multiLineBlockquote, // true (default)
          multiLineHeader: multiLineHeader, // true (default)
          multiLineCodeblock: multiLineCodeblock, // true (default)
          multiLineParagraph: multiLineParagraph, // true (default)
          multiLineCustomBlock: multiLineCustomBlock, // true (default)
          sanitizerOptions: satitizerOptions, // null (default)
          converterOptions: OpConverterOptions(
            customCssStyles: customCssStyles ??
                (DeltaInsertOp op) {
                  if (op.isImage()) {
                    final OpAttributes attributes = op.attributes;
                    final String? attrs = attributes['style'];
                    return <String>[
                      'max-width: 100%',
                      'object-fit: contain', 
                      '${attrs != null ? attrs : ''}'
                    ];
                  }
                  if (op.attributes['wiki'] != null) {
                    return <String>['$docLinksTag: ${op.attributes['wiki']}'];
                  }
                  if (op.attributes['line-height'] != null) {
                    return <String>['$docLineSpacingTag: ${op.attributes['line-height']}'];
                  }
                  return null;
                },
            inlineStyles: InlineStyles(<String, InlineStyleType>{
              'font': InlineStyleType(fn: (String value, _) => defaultInlineFonts[value] ?? 'font-family: $value'),
              'size': InlineStyleType(fn: (String value, _) {
                double? size = double.tryParse(value);
                if (size != null) return 'font-size: ${value}';
                if (value.equals('small')) return 'font-size: 11';
                if (value.equals('large')) return 'font-size: 15.5';
                if (value.equals('huge')) return 'font-size: 18';
                return null;
              }),
              'indent': InlineStyleType(fn: (String value, DeltaInsertOp op) {
                final double indentSize = (double.tryParse(value) ?? double.nan) * 3;
                final String side = op.attributes['direction'] == 'rtl' ? 'right' : 'left';
                return 'padding-$side:${indentSize}px';
              }),
              'list': InlineStyleType(map: <String, String>{
                'checked': "list-style-type:'\\2611';padding-left: 0.5em;",
                'unchecked': "list-style-type:'\\2610';padding-left: 0.5em;",
              }),
            }),
            inlineStylesFlag: true,
          ));
```

**Comment by dsyrstad** [comment]
@CatHood0 When using the `example` app in this repo (which uses `flutter_quill`), I get the following:

Editor:
![image](https://github.com/VisualSystemsCorp/vsc_quill_delta_to_html/assets/1719665/e044f4be-9b30-4793-b761-bebecef00888)

Delta JSON:
```json
[
  {
    "insert": "Should justify the text."
  },
  {
    "insert": "\n\n",
    "attributes": {
      "align": "justify"
    }
  }
]
```
HTML:
```html
<p style="text-align:justify">Should justify the text.</p><p style="text-align:justify"><br/></p>
```
This appears to be about the same HTML you expected.
![image](https://github.com/VisualSystemsCorp/vsc_quill_delta_to_html/assets/1719665/94ea1de3-a113-4ad2-ad31-1d896041fd75)

Note: To see the effects of the justification, you have to have more text, so I tried:
![image](https://github.com/VisualSystemsCorp/vsc_quill_delta_to_html/assets/1719665/aab2291d-af40-4c85-99f6-33cea12aa4c0)

Delta:
```json
[
  {
    "insert": "Should justify the text. Justification requires a paragraph long enough to exceed the line length so you can see the results of the justification. Test test test test test test test test test test test test test test test test test test test test test test."
  },
  {
    "insert": "\n\n",
    "attributes": {
      "align": "justify"
    }
  }
]
```

HTML:
```html
<p style="text-align:justify">Should justify the text. Justification requires a paragraph long enough to exceed the line length so you can see the results of the justification. Test test test test test test test test test test test test test test test test test test test test test test.</p><p style="text-align:justify"><br/></p>
```

Rendered HTML:
![image](https://github.com/VisualSystemsCorp/vsc_quill_delta_to_html/assets/1719665/63b39ba4-1d5d-41ff-9d44-57586a8f68fe)

 In summary, it appears to me that it is working. Please try it in the example app to see if it is working for you. If it works for you in the example app, but not in your app, there might be a configuration difference. If you have a different example, please reply to this issue.

**Comment by CatHood0** [comment]
You're right. The error comes from the package version in my pubspec (was 1.0.1). Thanks for the help
